client/src/components/Sub_RatingsReviews/RatingBreakdown.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default function RatingBreakdown({
+  reviewMetadata, totalAmtOfReviews, toggleSelectedRating, selectedRatings,
+}) {
+  const progress = Object.entries(reviewMetadata.ratings)
+    .map((entry) => [entry[0], entry[1] / totalAmtOfReviews, entry[1]]);
+
+  const currentFilters = Object.entries(selectedRatings)
+    .filter((entry) => !!entry[1])
+    .map((entry) => entry[0]);
+
+  return (
+    <div className="rating-breakdown">
+      {progress.map((entry) => {
+        const percentage = Math.round(entry[1] * 100);
+        return (
+          <div
+            key={entry[0]}
+            className="rating-container"
+            onClick={() => toggleSelectedRating(entry[0])}
+          >
+            <div className="rating-bar-label">{entry[0]} stars:</div>
+            <progress
+              className="rating-bar"
```

Comment:
i'm a little confused here. progress is not a react component, how can you do <progress /> and assign a classname to it?

Reply:
hi Quanjing, I was referring to this doc.

https://beta.reactjs.org/reference/react-dom/components/progress

It looks like react does consider it a component, but I _did_ have some issues yesterday styling it, so I think there's some weird behavior that I still have to figure out with this element type.

Reply:
interesting...